python/mxnet/callback.py

```diff
@@ -134,26 +134,35 @@ def __call__(self, param):
 
 
 class ProgressBar(object):
-    """Show a progress bar.
+    """Displays the progress of no. of batches processed
```

Comment:
Please change to :

"Displays a progress bar, indicating the percentage of batches processed within each epoch."